sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/statsEstimation/BasicStatsEstimationSuite.scala

```diff
@@ -283,14 +326,17 @@ class BasicStatsEstimationSuite extends PlanTest with StatsEstimationTestBase {
   private def checkStats(
       plan: LogicalPlan,
       expectedStatsCboOn: Statistics,
-      expectedStatsCboOff: Statistics): Unit = {
-    withSQLConf(SQLConf.CBO_ENABLED.key -> "true") {
+      expectedStatsCboOff: Statistics,
+      extraConfigs: Map[String, String] = Map.empty): Unit = {
+
```

Comment:
nit: remove this unnecessary change

Reply:
I am not sure I understand you here. Do we need to directly put the histogram configs inside this method? By default histogram is disabled and number of bins default value is 254.

Reply:
Yes, removed the extra line